packages/@aws-cdk/core/lib/cfn-fn.ts

```diff
@@ -172,6 +172,15 @@ export class Fn {
     return new FnFindInMap(mapName, topLevelKey, secondLevelKey).toString();
   }
 
+  /**
+   * 
+   * @param macroNameParameters The name of the macro to perform the processing, and the Parameters to pass to it
+   * @returns a token represented as a string
+   */
+  public static transform(macroNameParameters: string): string {
```

Comment:
Same comment here as in the `FnTransform` class (this is not the correct way to model this function).